### PR TITLE
Project/entry review

### PR DESCRIPTION
**Pull request by thenav56**

### CODE REVIEW

File: apps/notification/tasks.py [hunk 1]
```diff
@@ -24,3 +27,50 @@ def send_entry_comment_email(user_id, comment_id):
...
+            'CommentType': CommentType,
+            'notification_type': notification_type,
+            'comment': comment,
+        },
+        subject_template_name='entry/review_comment_notification_email.txt',
+        email_template_name='entry/review_comment_notification_email.html',
+    )
+
+
+def send_notifications_for_commit(comment_pk, meta):
```
Comment (k9845) [inline apps/notification/tasks.py]: `send_notifications_for_comment` might be better, what say?

File: apps/project/models.py [hunk 2]
```diff
@@ -365,6 +378,8 @@ class ProjectMembership(models.Model):
         null=True, blank=True, default=None,
         related_name='added_project_memberships',
     )
+    # Represents additional permission like QA
+    badges = ArrayField(enum.EnumField(BadgeType), default=list, blank=True)
```
Comment (k9845) [inline apps/project/models.py:382]: This must be removed in favour of a later one or is it intended?

Comment (thenav56) [reply apps/project/models.py:382]: Both are required, `ProjectMembership` is a lower instance and `ProjectUserGroupMembership` is an instance that creates multiple `ProjetMembership` instances.

Comment (k9845) [reply apps/project/models.py:382]: Okay, we better add a comment for these cases .

File: apps/notification/tasks.py (hunk 1 above)
Comment (thenav56) [reply apps/notification/tasks.py]: Good Catch

File: apps/project/models.py (hunk 2 above)
Comment (thenav56) [reply apps/project/models.py:382]: It should be self-explanatory. But added a comment as well.

**Review by k9845 — DISMISSED** [review]

**Pull request merged**